.travis.yml

```diff
@@ -7,7 +7,8 @@ matrix:
     include:
     - rust: nightly
       script:
+      - cargo install --git https://github.com/starkat99/cargo-when.git
       - cargo test --verbose
       - cargo test --no-default-features --verbose
-      - cargo test --features serde-1
+      - cargo when --channel nightly test --features serde-1
```

Comment:
I don't think this should be strictly necessary since this is in the travis 'matrix' which only runs rust nightly- but I can test to make sure after merging.